config.xhgui.yaml

```diff
@@ -0,0 +1,21 @@
+#ddev-generated
+hooks:
+  post-start:
+  - exec: |
+      cat <<-PHP > .ddev/xhprof/xhprof_prepend.php
+      <?php
+      // DDEV's built in xhprof handler breaks our own.
+      // We'll temporarily override it here, but return control back later.
+      // If you don't want this behavior, comment out the hooks in ".ddev/config.xhgui.yaml".
+      return;
+      PHP
```

Comment:
This line would be easier to understand if it were just the traditional `EOF` or maybe `PHP_EOF`. Caught me by surprise. Not that important of course.